### PR TITLE
add tests and fix the wrong calculation of Mean

### PR DESCRIPTION
**Pull request by szuecs**
Mean was computed as median.

### CODE REVIEW

**Comment by szuecs** [comment]
Try to retrigger CI

File: aspects/request_time_test.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+// func Test_RequestTimerHandler(t *testing.T) {
+// 	gin.SetMode(TestMode)
+// 	router := gin.New()
+// 	aspect := ginmon.newTestNewRequestTimeAspect(1.0, 2.0, 3.0)
+// 	router.Use(ginmon.RequestTimeHandler(aspect))
+// 	tryRequest(router, "GET", "/")
+// 	if assert.Equal(t, cnt.Count, 2, "Incrementation of counter does not work, expect %d but got %d %s", 2, cnt.Count, ballotX) {
+// 		t.Logf("RequestTimeHandler works, expect %d and got %d %s", 2, cnt.Count, checkMark)
+// 	}
+// }
```
Comment (mikkeloscar) [inline aspects/request_time_test.go]: Maybe this should be removed instead of commented out? :)

Comment (szuecs) [reply aspects/request_time_test.go]: Thanks!

**Comment by mikkeloscar** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**